docs/pages/messages/missing-root-task-in-turbo-json.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Missing root task in turbo.json
+description: Missing root task in turbo.json error
+---
+
+# Missing root task in turbo.json
+
+## Why This Error Occurred
+
+We call the scripts defined in the monorepo's root `package.json` the root tasks. These tasks often call `turbo`
```

Comment:
Thinking it might help to invert the phrasing here, e.g. `"Root tasks" are the scripts defined in your workspace's root package.json.` The first time I read it, it felt awkward.